server/src/main/java/org/elasticsearch/search/DocValueFormat.java

```diff
@@ -48,6 +49,8 @@
 
 /** A formatter for values as returned by the fielddata/doc-values APIs. */
 public interface DocValueFormat extends NamedWriteable {
+    long MASK_2_63 = 0x8000000000000000L;
+    BigInteger BIGINTEGER_2_64_MINUS_ONE = BigInteger.ONE.shiftLeft(64).subtract(BigInteger.ONE); // 2^64 -1
```

Comment:
These variables should be final and static ?

Reply:
@jimczi Thanks Jim, I get a hint from Intellij that `final static` declarations are unnecessary here, as interface variables are by default `final` and `static`.

Reply:
oh that's an interface, right. Sorry for the noise